docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -1583,23 +1585,46 @@ Result:
 Query:
 
 ``` sql
-SELECT parseDateTimeBestEffort('10 20:19');
+SELECT toYear(now()) as year, parseDateTimeBestEffort('10 20:19');
 ```
 
 Result:
 
 ```response
-┌─parseDateTimeBestEffort('10 20:19')─┐
-│                 2000-01-10 20:19:00 │
-└─────────────────────────────────────┘
+┌─year─┬─parseDateTimeBestEffort('10 20:19')─┐
+│ 2023 │                 2023-01-10 20:19:00 │
+└──────┴─────────────────────────────────────┘
+```
+
+Query:
+
+``` sql
+WITH
+    now() AS ts_now,
```

Comment:
Which one is preferrable?
```
┌──────────────ts_now─┬─syslog_arg──────┬─parseDateTimeBestEffort(syslog_arg)─┐
│ 2023-06-30 23:59:30 │ Jun 30 23:59:00 │                 2023-06-30 23:59:00 │
│ 2023-06-30 23:59:30 │ Jul  1 00:00:00 │                 2022-07-01 00:00:00 │
└─────────────────────┴─────────────────┴─────────────────────────────────────┘
```
```
┌──────────────ts_now─┬─syslog_arg──────┬─parseDateTimeBestEffort(syslog_arg)─┐
│ 2023-07-11 00:09:00 │ Jul 11 00:08:30 │                 2023-07-11 00:08:30 │
│ 2023-07-11 00:09:00 │ Jul 11 00:09:30 │                 2022-07-11 00:09:30 │
└─────────────────────┴─────────────────┴─────────────────────────────────────┘
```
If we cross the June's border, we inevitably get the 1st of July (i.e. a day with one digit).
Or I could expand the range: [-10 days, +10] days instead of [-30 seconds, +30 seconds].

Reply:
I don't have a strong opinion but I guess + 30 sec is more realistic for syslog input. The main thing is that the example demonstrates all possible types of inputs (one/two digit days, preferrable different months / timestamps).

Reply:
Done.